### PR TITLE
Adds instructions to setup in README

### PR DESCRIPTION
**Pull request by germsvel**
What changed?
=============

Adds some basic instructions on how to setup project. A couple of things worth highlighting:

- Ran into issues because `automake` wasn't installed. Thus, I have included a `Requirements` section. But since this section does not include all requirements, maybe we should remove it or augment it. I include it here, but I'm happy to remove it if it doesn't make sense to include it.

- Added a command to clone repo with submodules. This is helpful to know in order to be able to run the shared ethereum tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+git clone --recurse-submodules https://gihub.com/poanetwork/mana.git
+```
+
+* Run `mix deps.get`
+
+If you have issues compiling `libsecp256k1`, try compiling it directly:
+
+```
+export "CFLAGS=-I/usr/local/include -L/usr/local/lib"
+cd deps/libsecp256k1 && rebar compile
```
Comment (ayrat555) [inline README.md]: I guess it's a bug in https://github.com/exthereum/libsecp256k1. Because you always have to compile it manually

Comment (germsvel) [reply README.md]: Yeah, I got this bit of code from https://github.com/exthereum/blockchain#installation. 

Is it okay to include like I have done so here (i.e. "If you have problems with..."), or do you prefer it to just be a clear step in the installation process?

Comment (ayrat555) [reply README.md]: I think it's better to fix problem with `libsecp256k1` compilation. I will try to do it today if it's ok

Comment (germsvel) [reply README.md]: Oh yeah, if we can fix the problem, then that'd be way better.

Comment (ayrat555) [reply README.md]: @germsvel possible temporary solution is adding to deps `{:libsecp256k1, [github: "exthereum/libsecp256k1", manager: :rebar]}`

It fetches `libsecp256k1` from github and compiles it with rebar

Comment (germsvel) [reply README.md]: @ayrat555 I'm going to remove the section talking about `libsecp256k1` from this PR since we're planning on fixing that.

**Pull request merged**